zigzag/data/configs/zigzag-config-example.json

```diff
@@ -0,0 +1,5 @@
+{
+    "test_cycle": "test_cycle_name",
```

Comment:
Does this 'test_cycle' config overlap with the CLI option `--qtest-test-cycle` ?  If so which one is authoritative?